support/cas-server-support-cassandra-ticket-registry/src/main/java/org/apereo/cas/config/CassandraTicketRegistryConfiguration.java

```diff
@@ -0,0 +1,51 @@
+package org.apereo.cas.config;
+
+import org.apereo.cas.TicketSerializer;
+import org.apereo.cas.configuration.CasConfigurationProperties;
+import org.apereo.cas.ticket.TicketCatalog;
+import org.apereo.cas.ticket.registry.CassandraTicketRegistryCleaner;
+import org.apereo.cas.ticket.registry.CassandraTicketRegistry;
+import org.apereo.cas.ticket.registry.CassandraTicketRegistryDao;
+import org.apereo.cas.logout.LogoutManager;
+import org.apereo.cas.serializer.JacksonJsonSerializer;
+import org.apereo.cas.ticket.registry.TicketRegistry;
+import org.apereo.cas.ticket.registry.TicketRegistryCleaner;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.annotation.EnableScheduling;
+
+/**
+ * @author David Rodriguez
+ *
+ * @since 5.2.0
+ */
+@Configuration("cassandraTicketRegistryConfiguration")
+@EnableScheduling
+@EnableConfigurationProperties({CasConfigurationProperties.class, CassandraProperties.class})
+public class CassandraTicketRegistryConfiguration {
+
+    @Autowired
+    private CassandraProperties cassandraProperties;
+
+    @Bean(name = {"ticketRegistry", "cassandraTicketRegistry"})
```

Comment:
Avoid bean aliasing. Let the method name define the bean name, like other config classes..